dotcom-rendering/src/web/layouts/NewsletterSignupLayout.tsx

```diff
@@ -0,0 +1,332 @@
+	} = CAPIArticle;
+
+	const adTargeting: AdTargeting = buildAdTargeting({
+		isAdFreeUser: CAPIArticle.isAdFreeUser,
+		isSensitive: CAPIArticle.config.isSensitive,
+		videoDuration: CAPIArticle.config.videoDuration,
+		edition: CAPIArticle.config.edition,
+		section: CAPIArticle.config.section,
+		sharedAdTargeting: CAPIArticle.config.sharedAdTargeting,
+		adUnit: CAPIArticle.config.adUnit,
+	});
+	const contributionsServiceUrl = getContributionsServiceUrl(CAPIArticle);
+
+	const palette = decidePalette(format);
+
+	return (
+		<>
+			<div data-print-layout="hide" id="bannerandheader">
+				<Stuck>
+					<ElementContainer
+						showTopBorder={false}
+						showSideBorders={false}
+						padded={false}
+						shouldCenter={false}
+					>
+						<HeaderAdSlot
+							isAdFreeUser={CAPIArticle.isAdFreeUser}
+							shouldHideAds={CAPIArticle.shouldHideAds}
+							display={format.display}
+						/>
+					</ElementContainer>
+				</Stuck>
+				<ElementContainer
+					showTopBorder={false}
+					showSideBorders={false}
+					padded={false}
+					backgroundColour={brandBackground.primary}
+					element="header"
+				>
+					<Header
+						edition={CAPIArticle.editionId}
+						idUrl={CAPIArticle.config.idUrl}
+						mmaUrl={CAPIArticle.config.mmaUrl}
+						supporterCTA={
+							CAPIArticle.nav.readerRevenueLinks.header.supporter
+						}
+						discussionApiUrl={CAPIArticle.config.discussionApiUrl}
+						urls={CAPIArticle.nav.readerRevenueLinks.header}
+						remoteHeader={CAPIArticle.config.switches.remoteHeader}
+						contributionsServiceUrl={contributionsServiceUrl}
+					/>
+				</ElementContainer>
+				<ElementContainer
+					showSideBorders={true}
+					borderColour={brandLine.primary}
+					showTopBorder={false}
+					padded={false}
+					backgroundColour={brandBackground.primary}
+					element="nav"
+				>
+					<Nav
+						nav={NAV}
+						format={{
+							...format,
+							theme: getCurrentPillar(CAPIArticle),
+						}}
+						subscribeUrl={
+							CAPIArticle.nav.readerRevenueLinks.header.subscribe
+						}
+						edition={CAPIArticle.editionId}
+					/>
+				</ElementContainer>
+
+				{NAV.subNavSections && (
+					<>
+						<ElementContainer
+							backgroundColour={palette.background.article}
+							padded={false}
+							element="aside"
+						>
+							<Island deferUntil="idle">
+								<SubNav
+									subNavSections={NAV.subNavSections}
+									currentNavLink={NAV.currentNavLink}
+									format={format}
+								/>
+							</Island>
+						</ElementContainer>
+						<ElementContainer
+							backgroundColour={palette.background.article}
+							padded={false}
+							showTopBorder={false}
+						>
+							<StraightLines
+								count={4}
+								cssOverrides={css`
+									display: block;
+								`}
+							/>
+						</ElementContainer>
+					</>
+				)}
+			</div>
+
+			{CAPIArticle.config.switches.surveys && (
+				<AdSlot position="survey" display={format.display} />
+			)}
+
+			<main>
+				<ContainerLayout
```

Comment:
Bit of a design question here but would this look better if there were a border on this container? You can set borders using `borderColour={brandLine.primary}`